modules/appeals_api/spec/fixtures/valid_200996_headers.json

```diff
@@ -6,5 +6,7 @@
   "X-VA-Birth-Date": "1969-12-31",
   "X-VA-File-Number": "987654321",
   "X-VA-Service-Number": "876543210",
-  "X-VA-Insurance-Policy-Number": "987654321123456789"
+  "X-VA-Insurance-Policy-Number": "987654321123456789",
+  "X-Consumer-Username": "va.gov",
+  "X-Consumer-ID": "some-guid"
```

Comment:
consumer-id is automatically added by Kong, but is consumer-username? (i.e. does this need to be in the documentation)

Reply:
they are both auto generated by kong.